pkg/odo/cli/cli.go

```diff
@@ -70,9 +70,6 @@ OpenShift Commands:{{range .Commands}}{{if eq .Annotations.command "openshift"}}
 Utility Commands:{{range .Commands}}{{if eq .Annotations.command "utility" }}
   {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}
 
-GUI Commands:{{range .Commands}}{{if or (eq .Annotations.command "gui") }}
-  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{ if .HasAvailableLocalFlags}}
```

Comment:
You will need to keep `{{ if .HasAvailableLocalFlags}}` at the end of the Utility command block